app/models/calagator/source.rb

```diff
@@ -38,10 +38,10 @@ class Source < ActiveRecord::Base
 
   # Create events for this source. Returns the events created. URL must be set
   # for this source for this to work.
-  def create_events!(opts={})
+  def create_events!
     save!
-    events = to_events(opts).select(&:valid?)
-    events.reject!(&:old?) if opts[:skip_old]
+    events = to_events.select(&:valid?)
+    events.reject!(&:old?)
```

Comment:
I think the symbol to proc version is a little more readable, but either way is fine with me.